.github/workflows/ci.yml

```diff
@@ -1,92 +1,20 @@
 name: Continuous Integration
 on:
   push:
-    branches: [main]
+    branches: [ "feature/next-gen" ]
   pull_request:
+    branches: [ "feature/next-gen" ]
 
 jobs:
-  full-build:
-    runs-on: macos-12
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Full Build
-        run: ./fullbuild.sh
-      - name: Smoke Test
-        run: ./smoketest.sh
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v4.4.1
-  xcframework:
-    runs-on: macos-12
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Print Swift version
-        run: swift --version
-      - name: Build xcframework
-        working-directory: ./SplunkRumWorkspace/SplunkRum
-        run: ./build_binaries.sh
-      - name: Upload xcframework
-        uses: actions/upload-artifact@v4.3.3
-        with:
-          name: SplunkOtel.xcframework
-          path: ./SplunkRumWorkspace/SplunkRum/xcframeworks/SplunkOtel.xcframework
-  saucelabs-prepare:
-    runs-on: macOS-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-      - name: Setup Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: '3.8'
-      - name: Build for Sauce Labs
-        run: |
-          BUILD_FOLDER="work/splunk-otel-ios/splunk-otel-ios/SauceLabsTests"
-          BUILD_PATH="Build/Products/Debug-iphonesimulator"
-          xcodebuild -showsdks
-
-          xcodebuild -workspace SplunkRumWorkspace/SplunkRumWorkspace.xcworkspace \
-            -scheme SauceLabsTestApp \
-            -configuration Debug \
-            -sdk iphonesimulator \
-            -derivedDataPath SauceLabsTests
-          zip ${GITHUB_WORKSPACE}/SauceLabsTestApp.zip $(find ~/$BUILD_FOLDER/$BUILD_PATH/SauceLabsTestApp.app -type f)
-      - name: Upload test app to Sauce Labs
-        id: sl-upload
-        run: |
-          cd /Users/runner/work/splunk-otel-ios/splunk-otel-ios/
-          item_id=$(curl -u "${{secrets.SAUCELABS_USER}}:${{secrets.SAUCELABS_KEY}}" --location \
-            --request POST 'https://api.us-west-1.saucelabs.com/v1/storage/upload' \
-            --form 'payload=@"SauceLabsTestApp.zip"' \
-            --form 'name="SauceLabsTestApp.zip"' \
-            --form 'description="'"$GITHUB_SHA"'"' | jq -r .item.id)
```

Comment:
Do we know that jq will be installed on the build machine, or if not do we have some requirements document that lists it?

Reply:
We don't know. The saucelabs tests are currently failing in the main branch, and we need to procure a team account and fix this pipeline. Hence, for now this is removed in feature/next-gen branch

Reply:
And now I notice it was a removal, not an addition... thanks.